dicom-archive/dicomSummary.pl

```diff
@@ -199,11 +248,20 @@
 exit $returnVal;
 
 ######################################################################### end main ####################
-=pod 
-################################################
-Read archive metadata from database
-################################################
-=cut 
+
+=pod
+
+=head3 read_db_metadata($StudyUID)
+
+Reads the information stored in the database for a given StudyUID.
+
+INPUT: the DICOM Study Instance UID (StudyUID)
+
+RETURNS: path to the file where the result of the query (i.e. the DICOM metadata
```

Comment:
Followup to the comment above:

RETURNS: the path of the file containing the metadata for the given StudyUID or `undef` if none is found.

Reply:
great! Thanks!